src/form/v2/birth/forms/print-certificate.ts

```diff
@@ -531,13 +534,128 @@ export const BIRTH_CERTIFICATE_COLLECTOR_FORM = defineActionForm({
       },
       fields: [
         {
-          id: 'collector.collect.payment.data',
+          id: 'collector.collect.payment.data.after365',
```

Comment:
Would an id like `collector.collect.payment.data.after365` make more sense? Since the late reg target is configurable.